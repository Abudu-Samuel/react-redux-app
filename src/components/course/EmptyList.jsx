import React from 'react';

const EmptyList = () => (
  <div>
    <p className="text-center"><i className="fa fa-exclamation-circle " /></p>
    <p className="text-center">There are no courses</p>
  </div>
);

export default EmptyList;