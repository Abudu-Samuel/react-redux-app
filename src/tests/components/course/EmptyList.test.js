/* eslint-disable */
import EmptyList from '../../../components/course/EmptyList';

describe('<EmptyList />', () => {
  it('render EmptyList component', () => {
    const wrap = shallow(<EmptyList />);
    expect(toJson(wrap)).toMatchSnapshot();
  });
});