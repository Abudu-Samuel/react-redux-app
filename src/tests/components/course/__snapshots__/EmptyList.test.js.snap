// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyList /> render EmptyList component 1`] = `
<div>
  <p
    className="text-center"
  >
    <i
      className="fa fa-exclamation-circle "
    />
  </p>
  <p
    className="text-center"
  >
    There are no courses
  </p>
</div>
`;
